### PR TITLE
Update cross-tenant-mailbox-migration.md

### PR DESCRIPTION
**Pull request by msbemba**
added a section with information on how to remove endpoints and organization relationships after migration

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9960

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 1]
```diff
@@ -381,6 +381,12 @@ Migration batch submission is also supported from the new [Exchange admin center
 
 Once the mailbox moves from source to target, you should ensure that the on-premises mail users, in both the source and target, are updated with the new targetAddress. In the examples, the targetDeliveryDomain used in the move is **contoso.onmicrosoft.com**. Update the mail users with this targetAddress.
 
+### Remove endpoints and organization relationships after migration
+
+Use the Remove-MigrationEndpoint(/powershell/module/exchange/remove-migrationendpoint) cmdlet to remove existing migration endpoints for source or destination servers after the migration is complete.
+
+Use the Remove-OrganizationRelationship (/exchange/sharing/organization-relationships/remove-an-organization-relationship#use-exchange-online-powershell-to-remove-an-organization-relationship) cmdlet to remove existing oraganization relationship for source or destination servers after the migration is complete.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
Use the Remove-OrganizationRelationship (/exchange/sharing/organization-relationships/remove-an-organization-relationship#use-exchange-online-powershell-to-remove-an-organization-relationship) cmdlet to remove existing oraganization relationships for source or destination servers after the migration is complete.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this Pr. Thanks

**Comment by yogkumgit** [comment]
@kelleyvice-msft Gentle reminder

**Pull request merged**